query/src/test/java/org/infinispan/query/directoryinteraction/IndexStoredIndexedCacheTest.java

```diff
@@ -48,7 +48,6 @@ protected void createCacheManagers() throws Throwable {
       .invocationBatching()
          .disable()
       .deadlockDetection()
```

Comment:
this: https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/configuration/cache/ConfigurationChildBuilder.java#L21